resources/views/navbar.blade.php

```diff
@@ -0,0 +1,51 @@
+
+        <nav class="navbar navbar-expand-md navbar-light bg-white shadow-sm">
```

Comment:
please remove some extra indentions

Reply:
Resolved in 09e5fe60d3de17d7bffc0db84fe9623b961bb6d6